src/ui/metadata_panel.py

```diff
@@ -535,16 +536,20 @@ def create_settings(self, name):
         default_author = "First Last"
         # default license
         default_license = "Redistributable under CCAL version 2.0 : see CAreadme.txt"
-
-        self.default_settings = [default_stl_dir, default_part_name,
-                                 default_part_dir, default_author,
-                                 default_license]
+        # default Log directory
+        default_log_dir = Util.path_conversion(str(Path.home()) + "/Documents")
```

Comment:
Thanks for finding a method that would return the home directory. Since all the path related tasks are handled by `src.util` I think it would be ideal to create a method that would return default_log_dir simply like `return str(Path.home() / "Documents")`